e2e/tests/functional/plugins/telemetryTable/telemetryTable.e2e.spec.js

```diff
@@ -45,6 +45,54 @@ test.describe('Telemetry Table', () => {
     await expect(rows).toHaveCount(50);
   });
 
+  test('auto scrolls to the top of the table on load', async ({ page }) => {
```

Comment:
should be scrolls to top if sort is descending, and scroll to bottom if sort is ascending. can we test both scenarios?

Reply:
YOU GOT IT!